### ISSUE TITLE
PermissionController:pageInfo is not resolvable

### ISSUE THREAD

**Issue by RomeroMsk** [userfrosting/UserFrosting#638]
I'm getting an error when clicking on permission name on "Permissions" page:
```
UserFrosting\Sprinkle\Admin\Controller\PermissionController:pageInfo is not resolvable
```
There is no `pageInfo` method in this controller.

**Comment by alexweissman** [comment]
It hasn't been implemented.  Actually, we should either remove that link entirely, or the page should show the roles for that permission.

**Comment by RomeroMsk** [comment]
Well, any of these options will be better than showing an exception. But why not to allow administrator to edit a permission? It will be more handy than editing via DB.

**Comment by alexweissman** [comment]
Permissions are "part of the application", not "part of the data".  It wouldn't make sense for _any_ users to modify permissions (even admins), because of how intimately they are tied into the code.  They should be determined during the development process, not day-to-day usage.

**Comment by alexweissman** [comment]
Ok, this is now implemented in 4.0.18.  The permission page will show permission info, and a list of users who have that permission along with the roles through which each one has it.
